src/CMakeLists.txt

```diff
@@ -29,24 +29,13 @@ endif()
 if (GIF_LIBRARY)
     target_link_libraries       (leptonica ${GIF_LIBRARY})
 endif()
-if (JPEG_LIBRARY)
-    target_link_libraries       (leptonica JPEG::jpeg)
-endif()
 if (JP2K_FOUND)
     target_link_libraries       (leptonica ${JP2K_LIBRARIES})
 endif()
-if (PNG_LIBRARY)
-    target_link_libraries       (leptonica PNG::png)
-endif()
-if (TIFF_LIBRARY)
-    target_link_libraries       (leptonica TIFF::libtiff)
-endif()
 if (WEBP_FOUND)
     target_link_libraries       (leptonica ${WEBP_LIBRARIES})
 endif()
-if (ZLIB_LIBRARY)
-    target_link_libraries       (leptonica ZLIB::zlib)
-endif()
+target_link_libraries           (leptonica JPEG::jpeg PNG::png TIFF::libtiff ZLIB::zlib)
```

Comment:
> nothing changed

Nothing where? PUBLIC means that dependency of the Leptonica should link jpeg, png, tiff and zlib too. I.e. Leptonica itself will build/install fine, but problems will appear on user's side when library used.